crates/astria-sequencer/src/app.rs

```diff
@@ -68,6 +68,9 @@ use crate::{
 /// The inter-block state being written to by the application.
 type InterBlockState = Arc<StateDelta<Snapshot>>;
 
+/// The maximum number of bytes allowed in sequencer action data.
+const MAX_BLOCK_SEQUENCE_DATA_SIZE: usize = 256_000;
```

Comment:
To make it clear what we are talking about.
```suggestion
const MAX_BLOCK_SEQUENCE_BYTES: usize = 256_000;
```
Alternatively `_LEN` because you construct this from the length of byte buffers (just going off of `Vec::len`).

Bikeshed: I wonder if we should move away from calling this `sequence` and just start calling it `rollup_data` or something like that because that's much clearer.

Reply:
discussed offline and updated name.

Will save the bike shedding on sequencer vs rollup data bikeshed for later date. This is correct choice given current naming.